docs/preview/README.md

```diff
@@ -0,0 +1,17 @@
+![logo](../images/logo/invictus.png)
+
+Framework for testing integration workflows in Microsoft Azure.
+
+# Installation
+
+Easy to install it via NuGet.
+
+```shell
+PM > Install-Package Invictus-Testing
```

Comment:
_Perhaps this remark is a bit late in the process, but still.._
Shouldn't the name be more specific and point towards Logic Apps, instead of `Invictus-Testing`.
Or wouldn't we add any other test-framework under the Invictus-umbrella?

Reply:
That's a fair point, maybe renaming to `Invictus.Testing.LogicApps` for now is best then?

Reply:
Yes, let's rename it while it isn't used yet.
You never know whether other test-frameworks would be added later on.

Reply:
➡️ https://github.com/invictus-integration/testing-framework/issues/44

Reply:
Ha, will close #43 😄